lib/Commention.php

```diff
@@ -79,7 +79,7 @@ public function sourceFormatted(string $anonymous = null): ?string
         if ($this->website()->isNotEmpty()) {
             $author = '<a href="' . $this->website() . '" rel="noopener noreferrer nofollow">' . $author . '</a>';
         } else {
-            $author = "<strong>{$author}</strong>";
+            $author = "<span>{$author}</span>";
```

Comment:
Looks good … but maybe we should give it a class name, in case someone wants to style it? Not sure, whether that’s a great idea, though …

Reply:
Thanks! Yes, it to be styleable is definitely the goal. It can be targeted in CSS using `.commentions-list-item h4 span`, so I'd personally rather avoid additional generic classes?

Reply:
If the selector is unambiguous, then I agree that we don’t need an additional class here.